Dockerfile

```diff
@@ -37,20 +31,6 @@ RUN apt-get update \
   swig \
   gobject-introspection
 
-RUN cd /usr/local/src \
-  && wget ${MOZJPEG_URL}/v${MOZJPEG_VERSION}.tar.gz \
-  && tar xzf v${MOZJPEG_VERSION}.tar.gz
-
-RUN cd /usr/local/src/mozjpeg-${MOZJPEG_VERSION} \
-  && aclocal \
-  && autoconf \
-  && autoheader \
-  && libtoolize \
-  && automake --add-missing \
-  && ./configure \
-  && make \
-  && make install
-
 # we must not use any packages which depend directly or indirectly on libjpeg,
 # since we want to use our own mozjpeg build
 RUN apt-get install -y \
```

Comment:
@benschwarz does this mean we don't need these either, since we're moving mozjpeg?